views/accuracy-leaderboard.pug

```diff
@@ -0,0 +1,39 @@
+.row#accuracy-board-card
+  .col.s12
+    table.bordered.centered
+      thead
+        tr
+          th Rank
+          th User
+          th Accuracy
+            
+      tbody
+        each s in leaderboard
+          if s.rank == 0
+            tr.col-default
+              td ...
+              td ...
+              td ...
+          else
+            if s.points != 0 && s.rank <= 3
+              if s.rank == 1
+                - rowclass = 'col-gold';
+              if s.rank == 2
+                - rowclass = 'col-silver';
+              if s.rank == 3
+                - rowclass = 'col-bronze';
+            else
+              - rowclass = 'col-default';
+            if s.id == userid
+              - rowclass += ' col-isuser';
+            tr(class=rowclass)
+              td #{s.rank}
+              td
+                img.responsive-img.circle(src='img/leaderboard_profile.png')
+                | #{s.fname + ' ' + s.lname.substring(0, 1) + '.'}
+              if s.totalAttemptsCount === 0
+                td 0.00%
+              else
+                td #{((s.correctAttemptsCount / s.totalAttemptsCount) * 100).toFixed(2)}%
```

Comment:
might not be a good idea to have math here, do you think this should be moved somewhere else

Reply:
Accuracy (same goes for the other calculations) aren't stored anywhere which is why I calculate them here. I can try to pass them through from when I'm assigning the rank.